integration_tests/test_build_package_version.py

```diff
@@ -0,0 +1,34 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import subprocess
+
+import integration_tests
+
+
+class BuildPackageVersionTestCase(integration_tests.TestCase):
+
+    def test_build_package_gets_version(self):
+        self.run_snapcraft('pull', 'build-package-version')
```

Comment:
This is good. It would be better if we could check that the version pulled is the right one. Do we have a way to do it?